crates/bevy_pbr/src/lightmap/mod.rs

```diff
@@ -71,7 +72,7 @@ pub const LIGHTMAP_SHADER_HANDLE: Handle<Shader> =
 ///
 /// If bindless textures aren't in use, then only a single lightmap can be bound
 /// at a time.
-pub const LIGHTMAPS_PER_SLAB: usize = 16;
+pub const LIGHTMAPS_PER_SLAB: usize = 4;
```

Comment:
Is there a reason for the 4 specifically or is it just a nice number that happens to still work because it's small enough??